src/utils/link-handler.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+function link({ LINK_TITLES }) {
+  return function handler(h, node) {
+    const props = {};
+    if (node.url) {
+      props.href = node.url.replace(/\.md/, '.html');
```

Comment:
It would be ideal to make the extension configurable and dependent on the extension in the request. This might require a bit of mangling as `VDOMTransformer` does not get the current request object, but I think it would increase reusability.